### DISCUSSION TITLE
Puppeteer and Playwright scraper not working

### DISCUSSION THREAD

**Discussion by 714Lemons** [FlowiseAI/Flowise#2943] [Q&A]
![Bildschirmfoto 2024-08-06 um 11 12 07](https://github.com/user-attachments/assets/3aee460b-e513-420c-9c9c-77bf394a25d4)

When i use the Cheerio Scraper everything works but i dont like the results so i wanted to try the Puppeteer and the Playwright scraper but then i get:

Error: vectorsService.upsertVector - TypeError: (intermediate value) is not iterable (cannot read property undefined)

Flowise version: 2.0.1 deployed using terraform on azure

Puppeteer loads forever and than fails, probably a timeout and Playwright fails faster. Maybe the browser downloads are missing?
It would be great if someone could test them and see if i made a mistake or if this is a bug.

**Comment by easablade** [comment]
Did you resolve this issue? I have the same with Playwright and Flowise 2.1.3.

**Comment by victorelcaminas** [comment]
The same is true here with Flowise 2.1.4, but only when using it from Docker.
